### DISCUSSION TITLE
Hichi Smartmeter an OpenDTU-OnBattery einbinden

### DISCUSSION THREAD

**Discussion by Forstgrund** [hoylabs/OpenDTU-OnBattery#621] [Q&A] [answered]
OpenDTU-OnBattery läuft soweit....nur schaffe ich es nicht meinen Hichi Smartmeter Lesekopf einzubinden. In den **Stromzähler Einstellunge**n habe ich bei der Url= http://192.168.178.46 eingetragen (IP vom Hichi) Beim JSON Pfad hab ich einiges probiert( zb StatusSNS/SML/Watt_Summe) aber nichts funktionierte....Wer kann mir da mal bitte helfen?

**Comment by spcqike** [comment]
die URL ist wahrscheinlich falsch. oder gibt der Hichi auf seine IP direkt ein verwertbaren JSON?

schau mal hier
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Power-Meter

**Reply by Forstgrund** [reply, marked as answer]
Es hat geklappt.... mit http://192.168.178.xx/cm?cmnd=status%208 im Browser konnte ich die Rohdaten auslesen. Dann muss man nur die Oberbegriffe suchen....bei mir war dann: StatusSNS/DVS7612/power

Vielen Dank an spcqike

**Comment by nikolaus-69** [comment]
Hallo, mein Hitchi ist per Usb am Pi angeschloßen. Die Kommunikation mit Homeassistant funktioniert über EDL gut. Aber wie Konfiguriere ich den OpenDtu damit die Daten dort Verwertbar sind?
grüße vom nikolaus

**Reply by schlimmchen** [reply]
Was ist EDL? Wenn die Daten in HomeAssistent landen, ist vermutlich irgendwo ein MQTT Broker im Spiel?

**Comment by nikolaus-69** [comment]
EDL21
https://www.home-assistant.io/integrations/edl21/
